.github/workflows/encryption-pvc-kms-ibm-kp/action.yml

```diff
@@ -12,13 +12,13 @@ inputs:
     required: true
 
 runs:
+  env:
```

Comment:
Was this change intended in your commit? It looks related to #9545 instead.

Reply:
Yes, you can ignore this change (I removed it).
The new integration test keeps [failing](https://github.com/rook/rook/runs/4808456061?check_suite_focus=true), so I tried to understand the reason for the failure.